frontend/spec/controllers/spree/orders_controller_spec.rb

```diff
@@ -68,6 +68,34 @@
             Spree.t(:please_enter_reasonable_quantity)
           )
         end
+
+        context "when quantity is empty string" do
+          it "should populate order with 1 of given variant" do
+            expect do
+              post :populate, params: { variant_id: variant.id, quantity: '' }
+            end.to change { user.orders.count }.by(1)
```

Comment:
This tests the association of the order to the current user. If I understand correctly, the change this introduces does not have anything to do with the user, so we can harden the spec by only specifying that `post`ing changes the `Spree::Order`.count.